woocommerce/woocommerce
# add tests for shipping zone regions

### All Submissions:






### Changes proposed in this Pull Request:
Add tests for shipping zone regions as tests specific to addition, update and deletion of regions do not exist in the suite of api-core-tests



Closes https://github.com/woocommerce/woocommerce/issues/32371 .

### How to test the changes in this Pull Request:

1. Navigate to woocommerce/packages/js/api-core-tests
2. Execute npm run e2e:api
3. Tests should pass successfully

### Other information:




### FOR PR REVIEWER ONLY:
